src/main/java/com/devonfw/demoquarkus/service/AnimalRestService.java

```diff
@@ -1,4 +1,4 @@
-package com.devonfw.demoquarkus.rest.v1.controller;
+package com.devonfw.demoquarkus.service;
```

Comment:
Wouldn't it make sense to keep the version information (`com.devonfw.demoquarkus.service.v1`) as described here  https://github.com/devonfw/devon4j/blob/master/documentation/guide-service-layer.asciidoc#versioning?

Reply:
I think we should keep it simple ( without a version like other samples mtmj, jtq) unless we want to demonstrate the versioning. @hohwille any thought?